### PR TITLE
refactor: change temp filenames to correct and added navigation funct…

### PR DESCRIPTION
**Pull request by JoshM0101**
…ions to buttons

# Pull Request Review Comments

## Overview
- **Purpose of the PR**:
    - Updated Directory Names from temp_ to correct names
    - Change the path names according to their screen names
    - Implemented navigation for all the 4 buttons in each screens
    - Updated general_words_jp and en with new words, "ユーアー" and "users"
    - Updated the labels of the 4 buttons in all new screens

### CODE REVIEW

File: client/src/translations/general_words_jp.json [hunk 1]
```diff
@@ -67,5 +67,6 @@
     "usersValidationText3": "パスワードは8文字以上で、大文字、小文字、数字、特殊文字をそれぞれ1つ以上含めてください",
     "usersValidationText4": "有効なメールアドレスを入力してください ",
     "confirmEmailAddress": "メールアドレス確認",
-    "dateJoined": "入社日"
+    "dateJoined": "入社日",
+    "users": "ユーアー"
```
Comment (edward-stalley-work) [inline client/src/translations/general_words_jp.json:71]: Please change to the following!
ユーアー 　　→　　ユーザー

**Comment by edward-stalley-work** [comment]
lets use an 's' for all the tabs.

So the screen should display : clients, employees, business divisions and users.

I also noticed 'project' tab in original screens is missing an s.

The other tabs have an 's' - employee expenses etc.

**Comment by edward-stalley-work** [comment]
I am getting an error in my code where the sidebar is imported.
It looks like it has to do with upper and lower casing: sideBar and sidebar.

> File name '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/components/Sidebar/Sidebar.tsx' differs from already included file name '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/components/SideBar/Sidebar.tsx' only in casing.
  The file is in the program because:
    Imported via '../../components/SideBar/Sidebar' from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/Dashboard/Dashboard.tsx'
    Imported via "../../components/SideBar/Sidebar" from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/NotFound/NotFound.tsx'
    Imported via "../../components/Sidebar/Sidebar" from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/Projects/ProjectsListAndEdit.tsx'
    Imported via '../../components/Sidebar/Sidebar' from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/Projects/ProjectsRegistration.tsx'
    Imported via '../../components/Sidebar/Sidebar' from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pag

Screenshot 2024-09-19 at 15 29 57

**Review by edward-stalley-work — COMMENTED** [review]
Thank you for your hard work!
Please see comments when you have time!

File: client/src/translations/general_words_jp.json (hunk 1 above)
Comment (JoshM0101) [reply client/src/translations/general_words_jp.json:71]: I will change this immediately Ed.

**Comment by JoshM0101** [comment]
> I am getting an error in my code in a few of the pages where the sidebar is imported. It looks like it has to do with upper and lower casing: sideBar and sidebar.
> 
> > File name '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/components/Sidebar/Sidebar.tsx' differs from already included file name '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/components/SideBar/Sidebar.tsx' only in casing.
> > The file is in the program because:
> > Imported via '../../components/SideBar/Sidebar' from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/Dashboard/Dashboard.tsx'
> > Imported via "../../components/SideBar/Sidebar" from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/NotFound/NotFound.tsx'
> > Imported via "../../components/Sidebar/Sidebar" from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/Projects/ProjectsListAndEdit.tsx'
> > Imported via '../../components/Sidebar/Sidebar' from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pages/Projects/ProjectsRegistration.tsx'
> > Imported via '../../components/Sidebar/Sidebar' from file '/Users/impl/Documents/work_projects/datt_japan/bop/client/src/pag
> 
> Screenshot 2024-09-19 at 15 29 57

I will fix this right away Ed.

**Comment by JoshM0101** [comment]
> lets use an 's' for all the tabs.
> 
> So the screen should display : clients, employees, business divisions and users.
> 
> I also noticed 'project' tab in original screens is missing an s. Let's update it as well. project → projects
> 
> The other tabs have an 's' - employee expenses etc.

Noted Ed.

**Pull request merged**